src/app/analytics/services/analytics.service.ts

```diff
@@ -403,9 +390,10 @@ const analyticsService = {
   identify,
   identifyUsage,
   identifyPlan,
-  trackSignOut,
-  trackSignIn,
-  trackSignUp,
+  rudderanalyticsSignIn,
```

Comment:
Why are we bound by the analytics library?  The action is the same regardless of what library we are using rudder or any other. I would keep track+actionTotrack as the naming convention @sg-gs